public/server/db.json

```
@@ -185,10 +186,48 @@
   "fgs": {
     "definition": "For God's sake"
   },
+  
+  "tysm": {
+    "definition": "Thank You So Much"
+  },
+
+  "ty": {
+    "definition": "Thank You"
+  },
+
+  "syl": {
+    "definition": "See You Later"
+  }, 
+
+  "hbty": {
+  	"definition": "Happy Birthday To You"
+  }, 
+
+  "hbu": {
+  	"definition": "How About You"
+  }, 
+
+  "imy": {
+  	"definition": "I Missed You"
+  }, 
+
+  "ilu": {
+  	"definition": "I Love You",
+  	"alternatives": "I Like You"
+  }, 
+
+  "sys": {
+  	"definition": "See You Soon"
+  }, 
+
+  "dygm": {
+  	"definition": "Did You Get Me"
```

I think "Do You Get Me" is more common.
```suggestion
  	"definition": "Do You Get Me"
```